GVFS/GVFS.Installer.Mac/uninstall_vfsforgit.sh

```diff
@@ -43,6 +45,23 @@ function UnInstallVFSForGit()
         eval $rmCmd || exit 1
     fi
     
+    # Unloading Service LaunchAgent for each user
+    # There will be one loginwindow instance for each logged in user, 
+    # get its uid (this will correspond to the logged in user's id.) 
+    # Then use launchctl bootstrap gui/uid to auto load the Service 
+    # for each user.
+    if [ -f "${LAUNCHAGENTDIRECTORY}/$SERVICEAGENTLAUNCHDFILENAME" ]; then
+        for uid in $(ps -Ac -o uid,command | grep -iw "loginwindow" | awk '{print $1}'); do
+            unloadCmd="sudo launchctl bootout gui/$uid ${LAUNCHAGENTDIRECTORY}/$SERVICEAGENTLAUNCHDFILENAME"
+            echo "$unloadCmd..."
+            eval $unloadCmd || exit 1
```

Comment:
We don't have to block on this, since I think we already have this bug on Windows, but what happens if another user has an active mount process/GVFS.Service when I uninstall VFS for Git? Do we cleanly shut things down? Is there an opportunity to handle this more gracefully on mac?

Reply:
`GVFS.Service` is gracefully unloaded. Created an [issue](https://github.com/Microsoft/VFSForGit/issues/978) for more graceful shutdown of `GVFS.Mount`.